### ISSUE TITLE
extractRawMirroredAlleleBFractions(): Error in setSignals(res, dh) : object 'res' not found

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/aroma.core#2]
`extractRawMirroredAlleleBFractions()` gives error

``` r
Error in setSignals(res, dh) : object 'res' not found
```

because

``` r
> extractRawMirroredAlleleBFractions.RawAlleleBFractions
function (this, ...)
{
    beta <- getSignals(this)
    dh <- abs(beta - 1/2)
    res <- setSignals(res, dh)
    class(res) <- c("RawMirroredAlleleBFractions", class(res))
    res
}
<environment: namespace:aroma.core>
```

**Comment by HenrikBengtsson** [comment]
It's probably a `res <- clone(this)` that is missing.

**Comment by HenrikBengtsson** [comment]
Fixed in commit 538fec5.
